lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -2709,7 +2719,8 @@ def get_vm_events(self, vm, eventTypeIdList):
         eventManager = self.content.eventManager
         return eventManager.QueryEvent(filterSpec)
 
-    def wait_for_customization(self, vm, poll=10000, sleep=10):
+    def wait_for_customization(self, vm, cus_timeout=3600, sleep=10):
+        poll = int(-(-cus_timeout // sleep))
```

Comment:
Does this suffice ? 

```suggestion
        poll = int(cus_timeout // sleep)
```

Reply:
Yeah, maybe everyone knows that to set the timeout value be multiple of sleep(10), like 600, but if someone set the timeout value to 568, then I'm considering to get the ceiling of the division result: int(-(-568 // 10)) = 57. I hope I did not think too much. Thanks.

Reply:
Ok, I just use the formal way. Thanks.